env0/resource_module.go

```diff
@@ -119,6 +113,30 @@ func resourceModule() *schema.Resource {
 				Optional:    true,
 				Default:     false,
 			},
+			"is_gitlab": {
```

Comment:
why not using conflictsWith for these as well?
GL can't be BBS for example, seems like a good validation

Reply:
sure!